alf/drivers/off_policy_driver.py

```diff
@@ -253,37 +249,30 @@ def _make_time_major(nest):
         for u in tf.range(num_updates):
             if mini_batch_size < batch_size:
```

Comment:
conditional expression `mini_batch_size < batch_size`  is a tensor  as `batch_size` is a tensor
Peharps we should avoid using a tensor as a bool value

Reply:
What is the problem of that?

Reply:
Somehow  that using a tensor as bool doesn't work in graph mode sometimes

Reply:
But this one is not a problem.